### PR TITLE
replace lazy_static with once_cell

### PR DESCRIPTION
**Pull request by pinjeff**
replacing the dependence on `lazy_static` with `once_cell`, this will ensure that variables are initialized when needed instead of startup time.

### CODE REVIEW

**Comment by rgwood** [comment]
Thanks for the PR.

> this will ensure that variables are initialized when needed instead of startup time

As I understand it, `lazy_static` behaves the same way as `once_cell` here. They both lazily initialize static variables. Am I misunderstanding that?

(I still think we should adopt `once_cell` because [it's a candidate for the stdlib](https://doc.rust-lang.org/std/cell/struct.OnceCell.html), but I want to make sure I understand the motivation for this PR)

**Comment by pinjeff** [comment]
did a little more research, and it turns out they both behave the same, sorry for that mistake

i still think it beneficial to move to `once_cell` because it offers a more flexible API

File: crates/nu-command/src/platform/ansi/ansi_.rs [hunk 1]
```diff
@@ -16,212 +16,802 @@ struct AnsiCode {
...
+        AnsiCode {
+            short_name: Some("g"),
+            long_name: "green",
+            code: Color::Green.prefix().to_string(),
+        },
+        AnsiCode {
+            short_name: Some("gb"),
+            long_name: "green_bold",
+            code: Color::Green.bold().prefix().to_string(),
+        },
```
Comment (rgwood) [inline crates/nu-command/src/platform/ansi/ansi_.rs:30]: It's difficult to review these changes, and the code is harder to read now that every `AnsiCode` takes up 5 lines. Can you change it back to 1 line per `AnsiCode`? You may be able to use `#[rustfmt::skip]`

Comment (pinjeff) [reply crates/nu-command/src/platform/ansi/ansi_.rs]: fixed it, thanks for the feedback

**Review by rgwood — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**